vast.conf

```diff
@@ -3,10 +3,10 @@ system {
 ; endpoint = "localhost:42000"
 
 ;; The file system path used for persistent state.
-; db-directory = "vast.db"
+; db-directory = "/var/lib/vast/"
```

Comment:
The commented-out values in the example configuration should reflect the default values, not platform-specific recommendations. As far as I can tell, these are now `"vast.db"` and `"vast.db/log"` for `system.db-directory` and `system.log-file` respectively.